internal/monitors/filesystems/filesystems.go

```diff
@@ -243,6 +243,20 @@ func (m *Monitor) Shutdown() {
 	}
 }
 
+// GetExtraMetrics returns additional metrics that should be allowed through.
+func (c *Config) GetExtraMetrics() []string {
+	if c.ReportInodes {
+		return []string{
+			"df_inodes.free",
```

Comment:
This seems like it overlaps with #741 .

Reply:
@jaysfx can you confirm this?

Reply:
This is filesystems plugin whereas #741 is df plugin but I guess they have same/similar metrics. Not sure if there's an easy way to share this though.

Reply:
df/filesystem should not be dependent on each other IMO

Reply:
Yeah I think that makes sense but the groupings should be the same since the metrics are basically duplicated. Maybe punt on filesystems plugin until the df one is merged so that they're both done the same way.